src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -62,6 +62,9 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
         std::string model_path = variants[0].as<std::string>();
         if (ov::util::ends_with(model_path, suffix.c_str())) {
             return true;
+        } else if (ov::util::directory_exists(model_path) && ov::util::file_exists(model_path + "/saved_model.pb") &&
```

Comment:
I think you should implement a sort of `path_join` routine for joining two strings and use different separators. For Windows it should be `\\` and `/` - others.

Reply:
/ - universal delimiter, all supported platforms, \\ - may cause an issue on non-Windows. But thanks for platform specific method, I'll try to use it.